### PR TITLE
Simplified multi-start

### PR DESCRIPTION
**Pull request by aarmey**
Multi-start is just using Sobol numbers to generate initial values. This seems to work correctly here. In practice you'll want to run the local search over more iterations, and probably increase the number of initial points explored.

### CODE REVIEW

File: src/Hill.jl [hunk 1]
```diff
@@ -45,20 +45,37 @@ end
...
 
     return Optim.minimum(results), Optim.minimizer(results)
 end
 
 
+function optimize_helper(f, g!, low::Vector, high::Vector, maxstep::Int)
+    sobolPoints = 10
+    s = SobolSeq(low, high)
+    skip(s, sobolPoints)
+
```
Comment (Farnazmdi) [inline src/Hill.jl:64]: Thank you so much. I will continue based on this.
One small question. These two functions have the same name. I think the second is calling the first one. I can change the name if it is okay.

**Review by Farnazmdi — APPROVED** [review]
Thank you so much.

File: src/Hill.jl (hunk 1 above)
Comment (aarmey) [reply src/Hill.jl:64]: Yep. Which one gets called depends on whether you provide an `x0` argument.

Comment (aarmey) [reply src/Hill.jl:64]: Changing the name is fine.

**Comment by Farnazmdi** [comment]
Could I merge this into my branch?

**Pull request merged**